docs/cloud/dbaas/index.md

```diff
@@ -14,6 +14,7 @@
 * [Command Line tools](cli.md)
 * [Using backups](backups.md)
 * [Database operations](operations.md)
+* [How to migrate a database to Pukki (DBaaS)](../../support/tutorials/pukki_data_migration.md)
```

Comment:
I gave you not an optimal suggestion in RC. It would probably be better if this line is added to `/cloud/dbaas/postgresql/index.md` since the tutorial is PostgreSQL specific.